lib/containers/container_images.pm

```diff
@@ -185,6 +186,32 @@ sub test_opensuse_based_image {
     }
 }
 
+sub verify_userid_on_container {
+    my ($runtime, $image) = @_;
+    record_info "host uid", script_output "echo \$UID";
+
+    record_info "root user inside", "process runs in the user's host uid";
+    my $cid = script_output "$runtime run -d --rm --name test1 $image sleep infinity";
+    assert_script_run "$runtime exec -it $cid id";
+    assert_script_run "$runtime exec -it $cid cat /proc/self/uid_map";
```

Comment:
Shouldn't we validate whether the UID/GID mapping works as it is set in `/etc/subuid` and `/etc/subgid`.

Reply:
or maybe validating value of `/proc/self/loginuid` wouldn't suffice over here ?

Reply:
**i do validation of the mapping** with ` validate_script_output "$runtime top $cid user huser", sub { /root\s+1000/ };` for example. i added those two lines to get some information and i thought that it wouldnt hurt to leave them. 
Regarding `/proc/1/*` i guess you can retrieve many info. But i dont know how important is to validate them(one by one or selectively).

Reply:
> **i do validation of the mapping** with ` validate_script_output "$runtime top $cid user huser", sub { /root\s+1000/ };` for example. i added those two lines to get some information and i thought that it wouldnt hurt to leave them.
Yes, I can see it, but what is the point of just `cat`-ing any `proc struct` ?
> Regarding `/proc/1/*` i guess you can retrieve many info. But i dont know how important is to validate them(one by one or selectively).
``
I am pointing to `/proc/1/loginuid` as `loginuid` is set only once and cannot be reset. Not to all content of a process.

Reply:
in my case it was used to debug. if the `validate_script_output` fails i will have some output to compare. it just my approach. But i believe that the validation exists. or do you want to add more?

Reply:
Yes, the validation is just under those lines.

Reply:
one idea is to remove them. is this ok?

Reply:
> one idea is to remove them. is this ok?

I vote for removal of the `cat`

Reply:
removed all except the last one. The reason is because this fails in TW. For this reason i added a softfail. i left one check that it can be removed later or whatever.